src/import-data/protected-data-utils.ts

```diff
@@ -45,13 +46,23 @@ export const isProtectedPfbSource = (pfbUrl: URL): boolean => {
   });
 };
 
+/**
+ * Determine if a TDR snapshot is considered protected data.
+ * */
```

Comment:
I thought this was a typo but I see line 23 does this too. What is the effect of writing `* */` vs `*/`? I was expecting to see `*/` here, as is done on line 58 below.

If this is a stylistic (vs functional) choice, I think we should chose and follow one convention for the terra-ui code.

Reply:
It is unintentional. I think some interaction of my typing pattern and my editor's autocomplete makes this keep happening.

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/pull/4322/commits/9a84663131955211a8cb6cf3f5829cd265242634.